dhttp/server_hijacked.go

```diff
@@ -0,0 +1,75 @@
+package dhttp
+
+import (
+	"context"
+	"net"
+	"net/http"
+	"sync"
+)
+
+type connContextKey struct{}
+
+// configureHijackTracking configures (mutates) an *http.Server to provide slightly better tracking
+// of Hijack()ed connections.  It returns a 'close' function that closes all active hijacked
```

Comment:
I had a hard time reviewing this because I don't know which `Hijack()` that is or what _exactly_ it does. I'm not sure that's a thing to be fixed, just a comment.